patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -118,6 +118,22 @@
 +        return 1.0;
 +    }
 +
++    /**
++     * If this method returns true, then chunks received by the client will
++     * have {@link net.minecraft.world.chunk.Chunk#resetRelightChecks} called
++     * on them, queuing lighting checks for all air blocks in the chunk (and
++     * any adjacent light-emitting blocks).
++     *
++     * Returning true here is recommended if the chunk generator used also
++     * does this for newly generated chunks.
++     *
++     * @return true if lighting checks should be performed
++     */
++    public boolean shouldCheckLighting()
```

Comment:
Although it is documented, maybe `shouldClientCheckLighting` would clarify the function of this method a bit better?